test/unit/lib/completeOriginalUrl.js

```diff
@@ -0,0 +1,13 @@
+const chai = require('chai');
+const completeOriginalUrl = require('../../../app/lib/completeOriginalUrl');
+
+const expect = chai.expect;
+
+describe('completeOriginalUrl', () => {
+  const reqMock = { hostname: 'beta.nhs.uk', originalUrl: '/find-a-pharmacy/results?location=ls' };
+  const mockOriginalUrl = 'https://beta.nhs.uk/find-a-pharmacy/results?location=ls';
```

Comment:
This isn't a mock, it is the expected output. Can the variable name be updated accordingly please?

Reply:
Done